content/blog/18-19/g2_blogpost.md

```diff
@@ -0,0 +1,95 @@
+die, we moeten het eerlijk toegeven, niet altijd goed gedocumenteerd is.
+Eens dan eindelijk de setup voltooid is, komt nog het moeilijkste deel:
+het selecteren van een issue die:
+
+- relevant is
+- binnen je niveau ligt
+- enigszins doenbaar is in de beperkte tijd die je kan vrijmaken
+
+
+## G2 - Gateway
+
+De oplossing? NOG een project natuurlijk! G2 wordt onze eigenste
+project-management hub. Het zal projectleiders de tools aanbieden om issues
+en branches op een logische manier te structureren. Projecten worden 
+recursief onderverdeeld in subprojecten, die elk op zich een logisch
+samenhangende collectie van issues en branches zijn. 
+
+Ja, dit kan nu ook al op zekere hoogte met zaken zoals tags op git(hub|lab). 
+De meerwaarde zit dan ook vooral in de andere richting.
+
+Nieuwe leden kunnen op een eenvoudige en gecentraliseerde manier de lopende
+projecten verkennen. Hierbij moeten ze niet steeds de repos doorspitten,
+maar kunnen ze zoeken op talen, frameworks en "soort" projecten (backend, 
+frontend, etc). 
+
+Leden met al enige pull requests achter de kiezen kunnen na een hiatus
+zich makkelijk herorienteren in hun favoriete projecten, en een 
+subproject uitkiezen dat hun aanspreekt.
+
+Het projectbestuur kan mensen met bovenstaande vraag makkelijk doorverwijzen,
+alsook een oogje houden op de status en populariteit van alle projecten.
+
+## G2 - Gamification 2
+
+Na een eerste versie met bovenvermelde functionaliteit, is het een logische
+uitbreiding om hieraan ook een opgefriste versie van onze alom geliefde
+gamification metrics toe te voegen. De concrete implementatie hiervan ligt nog
+niet vast, maar er is duidelijk een voorkeur om meer richting "badges en 
+achievements" te gaan in de plaats van een arbitraire score.
+
+## We want you
+
+G2 wordt een ambitieus project, met een geplande timeframe van ongeveer een jaar
+voor het gateway gedeelte. Alle hens aan dek dus! Gezien het altijd leuker
+en makelijker is om in een versgeplukt project in te stappen is het nu hét moment
+bij uitstek om je kans te grijpen en er van het begin bij te zijn. 
+
+Hieronder een korte beschrijving van de geplande tech stack. Hier kunnen zeker
+nog wijzigingen in komen, maar het is alvast hoe we zullen beginnen:
+
+**Dataopslag**
+
+SQL-based database. Effectieve variant nog nader te bepalen, maar er zal 
+hoogstwaarschijnlijk met een abstractielaag worden gewerkt waardoor dit een minder
+belangrijk detail wordt.
+
+**Backend**
+
+Web server in [Clojure][clojure], een functionele taal met een focus op 
+elegante multithreading. Hét perfecte excuus om je functionele spieren nog eens
+te laten rollen, of juist om je voor te bereiden op de onvermijdelijke Haskell
+lessen.
+
+**Front-end**
+
+Hier houden we het bewust simpel. We gaan niet voor een flashy react/vue/angular
+SPA (Single Page App). Simpele HTML templating dus, met een gezonde dosis CSS en
+een lichte garnituur van JavaScript. Ideaal voor beginners!
```

Comment:
Typescript is de shit though

Reply:
Zeker zeker, maar weer een extra build step...